metadata-ingestion/src/datahub/ingestion/source/lookml.py

```diff
@@ -382,7 +382,15 @@ def _load_model(self, path: str) -> LookerModel:
 
     def _construct_datalineage_urn(self, sql_table_name: str, connection: str) -> str:
         platform = self._get_platform_based_on_connection(connection)
-        return f"urn:li:dataset:(urn:li:dataPlatform:{platform},{sql_table_name},{self.source_config.env})"
+
+        if "." in platform:
+            platform_name, database_name = platform.lower().split(".")
```

Comment:
I think we want `maxsplit=1` here 
```suggestion
            platform_name, database_name = platform.lower().split(".", 1)
```